### ISSUE TITLE
googlePackage is undefined

### ISSUE THREAD

**Issue by kmichek** [sir-dunxalot/ember-google-charts#57]
Hi, since I've installed "ember-cli": "~3.10.1" and "ember-google-charts": "^1.7.0" and getting this error:
jQuery.Deferred exception: googlePackage is undefined renderChart
render-chart.js:18
In my component I use pie-chart:
{{pie-chart data=dailyStatus options=dailyStatusOptions responsiveResize=true}}
Do I need to update my component too, please? Maybe some more imports. But I get the same error when downloading the package and run tests/dummy (which runs Ember 3.3.2)
Thank you very much for your great package.

**Comment by nfc036** [comment]
I am also a user of this package. It seems, that Google has a change in its scripts. Due to this, the check for ```packagesAreLoaded``` (equal to ```google.charts.loader```) always returns true, so that the visualization scripts are never loaded, see here: https://github.com/sir-dunxalot/ember-google-charts/blob/1fbe038d7e4396cf98d95479a6b2922e285a9284/addon/services/google-charts.js#L19-L40

When changing this lines to 
```
  loadPackages() {
    const { google: { charts, visualization } } = window;

    return new RSVP.Promise((resolve, reject) => {
      if (visualization !== undefined) {
        resolve();
      } else {
        charts.load('current', {
          language: this.get('language'),
          packages: this.get('googlePackages'),
        });

        charts.setOnLoadCallback((ex) => {
          if (ex) { reject(ex); }

          resolve();
        });
      }
    });
  },
```
it seems to work again. However, I am not 100% sure, that this is the solution for your problem as well.

**Comment by nfc036** [comment]
Also, see this pull request: https://github.com/sir-dunxalot/ember-google-charts/pull/56

**Comment by kmichek** [comment]
Yes, I've modified google-charts.js as instructed and it works fine now. Thank you very much!

**Comment by gpfiel** [comment]
Hi,
I've got the same issue. Is it available an update with those changes above?

**Comment by EmberJSParis** [comment]
same issue here - I noticed that the pull requests has failed ( phil-mellor  and  nfc036 ) - is the author sir-dunxalot here ? Thanks -to summ =  visualization is not working anymore  in ember (ember install ember-google-charts)

**Comment by sir-dunxalot** [comment]
This issue should be fixed by #59.
